style.css

```diff
@@ -102,12 +102,109 @@ a {
   padding-left: 5%;
 }
 
+.displayNone {
+  display: none;
+}
+
+.modalDisplay {
+  display: inline-flex;
+  position: fixed;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(193, 199, 208, 0.6);
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  z-index: 2;
+}
+
+/* modal window classes */
+.cardDiv {
+  position: relative;
+  min-height: 95vh;
+  width: 90%;
+  background-color: white;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.closeModal {
+  color: white;
+  position: absolute;
+  top: 3%;
+  right: 7%;
+  font-weight: 600;
+}
+
+.width90 {
+  width: 90%;
+}
+
+.modalTitle {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  width: 90%;
+}
+
 .closeMenuSymbol {
   margin-left: 90%;
   margin-top: 5%;
   color: #67798e;
 }
 
+form ul {
+  margin-top: 58px;
+  margin-bottom: 46px;
+  display: flex;
```

Comment:
- [ ] It would be better if we implement the DRY principle, here we have repeated style rules which needs to be fixed. For example; `display:flex` has appeared more than once so we need to improved our CSS file. We can achieve this in many ways but I would highlight one way, kindly create a class for `display:flex` and call it maybe `.flex-display { display: flex}` and pass the class to any other elements with the display flex property

Reply:
Thanks for the suggestion. Already work in most of them, kept some inside the desktop media query.